app/dashboard/migrations/0039_auto_20180201_0958.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.0.1 on 2018-02-01 16:58
```

Comment:
@mbeacom when you write a migration to migraate the current fulfillments into the new structure.. youll need to make sure that it runs BEFORE this migration.

Reply:
on second thought... i think we might want to not run this migration quite yet.. i belive this will be important for the legacy functionality to have these database fields in tact

Reply:
I think we will need to do something like manually populate the `fulfillments` JSON field with existing fulfilled bounties.  Hopefully there aren't too many in the "fulfilled but not yet accepted" state.  The other states should be OK I think.

Reply:
@owocki Following our conversation on slack, I was going to create a schema migration to create the new fulfillment model and add the relation -> second migration (data) from old `bounty` fields to bountyfulfillment and add the relation -> third schema migration to delete the old fields.  Thoughts?

Reply:
sounds great